### ISSUE TITLE
Crash when opening a note with ordered/unordered lists

### ISSUE THREAD

**Issue by m-irigoyen** [Rolisteam/rolisteam#64]
[![Build Status](http://www.rolisteam.org/sites/default/files/pixture_reloaded_logo.png)](http://www.rolisteam.org)

#Your configuration:
Rolisteam version: 1.8.2
Operating System and version: 
-Windows 10, version 1703

Rolisteam Setting: 
- Unconnected (I just closed the connection window)

#What did you do ? (Explain each step from starting rolisteam to the issue)
- Start rolisteam
- File/Open/note : open attached note
- crash.
The attached note is a minimal repro case containing some ordered and unordered lists copy/pasted from an etherpad.
[test.zip](https://github.com/Rolisteam/rolisteam/files/1121499/test.zip)

#What did you expect to see ? (Make sentences and explains in details with words)
- Opening the same note I just saved

#What did you get instead ? (Describe in details the issue) 
- Immediate crash. Debug output from QtCreator shows "ASSERT: "editBlock > 0" in file text\qtextdocument_p.cpp, line 1186"

**Comment by m-irigoyen** [comment]
I don't think this actually comes from the Etherpad copy/paste : I just crashed again with that note 
[testNatif.zip](https://github.com/Rolisteam/rolisteam/files/1121530/testNatif.zip) which is a native rolisteam note with some bullet lists.
Creating, editing and saving the note is fine, it crashes on open. Same assert.

**Comment by obiwankennedy** [comment]
The text editor used by rolisteam is an old project. I integrated it in rolisteam but the code may be old.
I will have to investigate in code which isn't mine.

It will be interesting.

**Comment by m-irigoyen** [comment]
Hé, it always is in those cases :) Also I can confirm it doesn't do it with html file format.

**Comment by obiwankennedy** [comment]
I can't reproduce the crash on windows. I have to try on windows. My windows Pc is out of my reach (cause I'm on holidays). I will work on it next week.
